src/main.dart

```diff
@@ -1,8 +1,99 @@
-// TODO: Run task list app
-void main(List<String> args) {
-  print('Hello World');
+import 'dart:developer';
+
+import 'dart:io';
+
+class Task {
+  int? taskID;
+  String? name;
+  String? description;
+  bool? isCompleted;
+
+  Task(this.taskID, this.name, this.description, this.isCompleted);
+}
+
+class TaskManager {
+  List<Task> tasks = [];
+  int taskID = 1;
+
+  // Add task
+  void addTask(String? name, String? description) {
+    tasks.add(Task(taskID, name, description, false));
```

Comment:
Once you make the defailt constructor this will break. Use this instead:
```suggestion
    tasks.add(Task(
        id: taskID,
        name: name,
        description: description,));
```